src/klr.core45/tpa.cpp

```diff
@@ -6,78 +6,82 @@
 
 BOOL CreateTpaBase(LPWSTR** ppNames, size_t* pcNames, bool bNative)
 {
-    const size_t count = 32;
+    const size_t count = 34;
```

Comment:
I didn't add any new reference in this change but `tpa.cpp` is modified by build script. I can put this change in a separate commit. cc @troydai to confirm there is nothing wrong with this part.